pkg/fftypes/verifier.go

```diff
@@ -0,0 +1,44 @@
+// Copyright © 2022 Kaleido, Inc.
+//
+// SPDX-License-Identifier: Apache-2.0
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package fftypes
+
+// VerifierType is the type of an identity berifier. Where possible we use established DID verifier type strings
+type VerifierType = FFEnum
+
+var (
```

Comment:
I realize I still need to add the `GET` `/api/v1/namespaces/{ns}/identities/{idid}/did` export for a DID JSON object that is mentioned in the FIR. This code here was written with that in mind, but I used snake-case consistent lower-case values here as it's important in how we treat FFEnum values (these can be translated into the DID ones that are camel-case in the JSON mapping layer).